docs/05-technical-reference/03-architecture/assets/svls-architecture.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The new diagram style looks great!

however, at least on my small laptop screen, the font size is minuscule, especially in the legend and descriptions of the arrows:

![image](https://user-images.githubusercontent.com/76950046/122061499-dc7ca980-cdee-11eb-9d29-2c223656d1ab.png)

![squint](https://i.imgflip.com/45amwu.png)

Reply:
😆 Ok, so that's a question to everyone, as I totally followed guidelines on this. I will raise this concern 😉